src/helper-modules/dun-and-bradstreet/dun-and-bradstreet.service.ts

```diff
@@ -0,0 +1,56 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { DUN_AND_BRADSTREET } from '@ukef/constants';
+import { HttpClient } from '@ukef/modules/http/http.client';
+
+import { DunAndBradstreetConfig } from '@ukef/config/dun-and-bradstreet.config';
+import { createWrapDunAndBradstreetHttpGetErrorCallback } from './wrap-dun-and-bradstreet-http-error-callback';
+
+@Injectable()
+export class DunAndBradstreetService {
+  private readonly httpClient: HttpClient;
+  private readonly encoded_key: string;
```

Comment:
Should we use camelCase for all variables? encodedKey, accessToken

Reply:
Good spot, thanks I'll do that.